assets/src/stories-editor/components/media-inserter/index.js

```diff
@@ -118,6 +133,14 @@ const applyWithSelect = withSelect( ( select ) => {
 		},
 		// As used in <HeaderToolbar> component
 		showInserter: select( 'core/edit-post' ).getEditorMode() === 'visual' && select( 'core/editor' ).getEditorSettings().richEditingEnabled,
+		getCurrentPageAttributes: () => {
```

Comment:
Can't we just directly pass on the `mediaType` and `mediaId` attributes here instead of a function that returns all the attributes? `mediaType` only might even be enough.